src/Renderers/CodeNodeRenderer.php

```diff
@@ -83,6 +83,10 @@ public function render(): string
                 'languageMapping' => self::LANGUAGES_MAPPING[$language] ?? $language,
                 'code' => $code,
                 'lineNumbers' => rtrim($lineNumbers),
+                // this is the number of digits of the codeblock lines-of-code
+                // e.g. LOC = 5, digits = 1; LOC = 18, digits = 2
+                // this is useful to tweak the code listings according to their length
+                'numLocDigits' => (int) floor(log10(\count($lines))) + 1,
```

Comment:
i don't understand why using `log10()` here...
something like `strlen((string) \count($lines))` wouldn't be more readable?

Reply:
For this kind of things I always end to go to the pure math solution ... but your proposal is more easy to understand, so I changed this. Thanks!